ide/coqide/coqide.ml

```diff
@@ -1371,10 +1393,12 @@ let read_coqide_args argv =
     |"-coqtop" :: [] ->
       output_string stderr "Error: missing argument after -coqtop"; exit 1
     |"-debug"::args ->
-      Minilib.debug := true;
-      Flags.debug := true;
-      Exninfo.record_backtrace true;
+      set_debug ();
       filter_coqtop coqtop project_files bindings_files ("-debug"::out) args
+    |"-xml-debug"::args ->
+      set_debug ();
+      Flags.xml_debug := true;
+      filter_coqtop coqtop project_files bindings_files ("-xml-debug"::out) args
```

Comment:
This new option would be better in a separate PR I guess.

Reply:
Well, there are only 5 lines of code related to this option.  It was essential for me to do this PR.  It is only a debugging option.  Separating it into multiple PRs seems like creating more work for little, if any, benefit.  I don't think it's even worth a separate commit.  WDYT?

Reply:
Same answer as for merlin's file.